src/jvmMain/kotlin/io/github/fourlastor/wilds_launcher/extensions/ThrowableFullTraceExtension.kt

```diff
@@ -0,0 +1,7 @@
+package io.github.fourlastor.wilds_launcher.extensions
+
+fun Throwable.fullTrace(): String = """
+        $message
+        ${stackTraceToString()}
+        ${cause?.also { "Caused by: ${it.fullTrace()}" }}
```

Comment:
`also` doesn't return the new value, it only runs the function inside
```suggestion
        Caused by: ${cause?.fullTrace().orEmpty()}
```